tron/config/config_parse.py

```diff
@@ -67,7 +68,11 @@ def validator(value, config_context):
         )
 
         try:
-            value % context
+            if "%" in value:
```

Comment:
This is probably too generous. We have `%` all over the place.

Can we "just" support both? Where we do both types of string interpolation, regardless?

Reply:
I am ok to just support both here, but how could we decide which rendered command we should actually run? I guess we still need a condition to decide. If so, I think it's worth to keep it consistent between validation and action run.

Reply:
But why can't we literally do both?
Like format it in both ways. I don't think we need a condition to decide.
Also yes, however formatting happens it should be consistent between the actionrun and the validation.

Reply:
Ah, I see. That makes sense.